docs/designs/output.md

```diff
@@ -54,22 +66,38 @@ https://docs.microsoft.com/en-us/dotnet/api/system.string.html#Instantiation
 
 ## Content Outputs
 
-Each input file transforms to zero or more output files depending on its content type. `docfx` places output files in `output-path` relative to output directory. **Regardless of static rendering or dynamic rendering, output path shares the same schema**:
+Each input file transforms to zero or more output files depending on its content type. `docfx` places output files in `output-path` relative to output directory.:
+
+### Docs hosting output path schema:
 
 `{output-dir}/{base-path}/{moniker-hash}?/{site-path}`
 
 ```
   base-path moniker-hash    site-path (relative-to-base-path)
-      |--^-| |--^--| |----------------^----------|
+      |--^-| |--^--| |----------^----------|
+_site/dotnet/01ddf122/api/system.string.json
+```
+
+### Static hosting output path schema:
+
+`{output-dir}/{base-path}/{site-path}`
+
+```
+  base-path     site-path (relative-to-base-path)
+      |--^-| |------------------^-----------------|
 _site/dotnet/01ddf122/api/system.string/index.html
+
+  base-path     site-path (relative-to-base-path)
+      |--^-| |------------------^----------|
+_site/dotnet/01ddf122/api/system.string.html
```

Comment:
static hosting with ugly URL? since the detailed example has been added later. Do we still need it here?